NitroxServer/Banning/IpBanning.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NitroxServer.Banning
+{
+    public static class IpBanning
+    {
+        public static List<string> BannedIPs = new List<string>();
+        public static string BanFilePath { get; } = Path.GetFullPath(Path.Combine(Environment.GetEnvironmentVariable("NITROX_LAUNCHER_PATH") ?? "", "Player Bans.txt"));
```

Comment:
Maybe get the launcher path from [NitroxModel/Helper/NitroxAppData.cs.](https://github.com/SubnauticaNitrox/Nitrox/blob/master/NitroxModel/Helper/NitroxAppData.cs)